src/pages/benefits/benefitFormUI/BenefitFormUI.tsx

```
@@ -61,51 +92,44 @@ const BenefitFormUI: React.FC = () => {
-        });
-        const allSchema = {
-          ...applicationSchema,
-          required,
-          properties,
-        };
-
-        setFormSchema(allSchema);
+    const required = Object.keys(properties).filter((key) => {
+      const isRequired = properties[key].required;
+      if (isRequired !== undefined) {
+        delete properties[key].required;
```

Biome (1.9.4)

[error] Avoid the delete operator which can impact performance.

Unsafe fix: Use an undefined assignment instead.


(lint/performance/noDelete)